pincer/objects/user/user.py

```diff
@@ -110,7 +110,7 @@ class User(APIObject):
         Whether the email on this account has been verified
     """
 
-    id: Snowflake
+    id: Snowflake = None
```

Comment:
is this suppose to be None or MISSING?

Reply:
It's a placeholder so dataclasses doesn't tell at me for having a required field before an optional one. What do you think is better for that?

Reply:
I decided to make is MISSING